src/languages/golfscript/index.ts

```diff
@@ -63,10 +68,15 @@ const golfscriptLanguage: Language = {
   emitter: emitProgram,
   phases: [
     search(hardcode()),
-    required(printIntToPrint),
+    required(
+      printIntToPrint,
+      implicitlyConvertPrintArg,
```

Comment:
Is there a reason these are not applied after the seearch?

Reply:
only that they need to happen before `printToImplicitOutput`

Reply:
> only that they need to happen before `printToImplicitOutput`

Yes, I meant all of the 73-76.

Reply:
Then no not particularly, would there be benefit to moving them after the search?

Reply:
Just in general if you apply the required plugins last, I think it's easier to ensure you end up with emittable program and don't have to worry about what the search plugins are doing... Also if you perform too many languages specific transforms before the main search, you might prevent some language agnostic golfing to happen.
I consider the `printIntToPrint` to be different, since that's like a preprocessing step that's gonna be included in (almost) every lang so that the rest of Polygolf doesn't even need to know that `print_int` exists.

But here, I don't think it matters much.

Reply:
makes sense, I'll go ahead and shift them to after the search.